Introduction to compilers.playground/Pages/IRGeneration.xcplaygroundpage/Sources/DebuggerView.swift

```diff
@@ -115,7 +115,7 @@ public class DebuggerView: NSSplitView, NSTouchBarDelegate {
 
         // Create the header
         let header = NSTextField(labelWithString: "IR Debugger")
-        header.font = NSFont.systemFont(ofSize: 33, weight: NSFontWeightSemibold)
+        header.font = NSFont.systemFont(ofSize: 33, weight: NSFont.Weight.semibold)
```

Comment:
I think, we could simply use `.semibold` instead of the fully qualified name here.